config/routes.rb

```diff
@@ -290,6 +290,13 @@
   get '/css/donate-button.css' => 'widget#v1_css'
   get '/css/donate-button.v2.css' => 'widget#v2_css'
 
+  # Routes
+
+  get '/404', to: 'errors#not_found'
```

Comment:
Minor change: could these error routes be in numerical order? It'd be easier for maintenance.